docs/rewards/reward-boosts.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: reward-boosts
+title: Reward Boosts
+sidebar_label: Reward Boosts
+slug: /project/reward-boosts
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+### Overview
+
+**Reward boosts** are additional rewards given to devices that contribute weather data during daily distribution, added to their **base rewards**. Learn more about the daily rewards at the [Reward Mechanism (RM) ](/reward-mechanism) section.
+
+**Reward Boosts** rewards devices with extra $WXM tokens for meeting specific criteria. 
+
+So far the following **Reward Boosts** are sheduled to be implemented:
+
+### Beta Rewards
+
+According to the [Token Allocation table](https://weatherxm.network/docs/wxm-token.html#token-allocation), 3M $WXM are reserved to reward stations that participated to the network before the Token Launch Day ("Beta period").
+
+Beta rewards distribution will last 425 days, with Day 1 being the Token Launch day.
+
+#### Calculation:
+
+The following calculations will take place for each station, the day before the Token Launch:
+
+`RewardableBetaHours[s]`:
+This is the total number of hours the station provided valid data during the Beta period.
+A rewardable hour is an hour during which the station sent at least one valid weather data point.
+
+`StationBetaRewards[s] = 3000000 * RewardableBetaHours[s]/Sum(RewardableBetaHours of all stations)`:
```

Comment:
This syntax is somewhat confusing. I strongly suggest we go with Math formula syntax, as described here:
https://github.blog/2022-05-19-math-support-in-markdown/

The content could be take - and adapted - from here our internal outline wiki.